src/components/Footer/Footer.js

```diff
@@ -314,7 +315,7 @@ const styles = StyleSheet.create({
   container: {
     alignItems: 'center',
     borderBottomWidth: 1,
-    borderBottomColor: COLORS.gray
+    borderBottomColor: COLORS.gray200
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The border color has been changed from `COLORS.gray` to `COLORS.gray200`. Ensure that `COLORS.gray200` is defined and provides the desired visual effect.
```diff
- borderBottomColor: COLORS.gray
+ borderBottomColor: COLORS.gray200
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->